Adyen/Model/Checkout/Name.cs

```diff
@@ -79,63 +78,57 @@ public enum GenderEnum
         /// <summary>
         /// Initializes a new instance of the <see cref="Name" /> class.
         /// </summary>
-        /// <param name="FirstName">The first name. (required).</param>
-        /// <param name="Gender">The gender. &gt;The following values are permitted: &#x60;MALE&#x60;, &#x60;FEMALE&#x60;, &#x60;UNKNOWN&#x60;. (required).</param>
-        /// <param name="Infix">The name&#39;s infix, if applicable. &gt;A maximum length of twenty (20) characters is imposed..</param>
-        /// <param name="LastName">The last name. (required).</param>
-        public Name(string FirstName = default(string), GenderEnum Gender = default(GenderEnum), string Infix = default(string), string LastName = default(string))
+        /// <param name="firstName">The first name. (required).</param>
+        /// <param name="gender">The gender. &gt;The following values are permitted: &#x60;MALE&#x60;, &#x60;FEMALE&#x60;, &#x60;UNKNOWN&#x60;. (required).</param>
+        /// <param name="infix">The name&#39;s infix, if applicable. &gt;A maximum length of twenty (20) characters is imposed..</param>
+        /// <param name="lastName">The last name. (required).</param>
+        public Name(string firstName, GenderEnum gender, string infix, string lastName)
```

Comment:
infix is not required field so it can be string Infix = default(string) [you can check it here in docs](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/v52/payments__reqParam_shopperName)

Reply:
The problem with this is that you can't have required parameters _after_ optional parameters :-/ Adding a default to infix wouldn't compile without defaulting lastname as well.
Ideally, the ctor should be (firstname, lastname, gender, infix = null), but to adjust the order of parameters would be a very breaking change and could potentially trip up existing implementations.

Reply:
Yes, good catch. The costructor can be as you set it